lib/pharos/salt.rb

```diff
@@ -27,6 +27,11 @@ def self.minions
       JSON.parse(res.body)["return"].first
     end
 
+    # Call mine.update on salt for every minion specified by targets.
+    def self.mine_update(targets: "*")
```

Comment:
Do we want to `call` directly from the outside? I think it exposes too many salt information to users (the controller in this case). It would expose `tgt` and `fun`, I don't know if we want to expose that much.